app/components/DepositWithdraw/blocktrades/BlockTradesBridgeDepositRequest.jsx

```diff
@@ -16,6 +16,25 @@ import {Asset} from "common/MarketClasses";
 import {getConversionJson} from "common/gatewayMethods";
 import PropTypes from "prop-types";
 import {Modal} from "bitshares-ui-style-guide";
+import QueryString from "query-string";
+import ls from "common/localStorage";
+import {UserManager, WebStorageStateStore} from "oidc-client";
+
+let oauthBlocktrades = new ls("__oauthBlocktrades__");
+let oidcStorage = new ls(
```

Comment:
Why two storages?

Reply:
oidcStorage belongs to oidc library. It keeps refresh token and other data related to token. oauthBlocktrades is storage implemented to keep information if user shoudl be able to use token. oidcStorage should be enough to keep correct logic but I am not sure if oidc library can keep refresh toke after user removal what can cause unwanted loging users.

Reply:
I looked on code again and checking if 'refresh_token' is in oidc. Storage storage should be enough to decide about user login but it will be good to test it more.

Reply:
It would be preferable to use one storage here, if possible. Please do